huaweicloud/services/sfs/resource_huaweicloud_sfs_turbo.go

```diff
@@ -635,39 +638,47 @@ func resourceSFSTurboDelete(ctx context.Context, d *schema.ResourceData, meta in
 	return nil
 }
 
-func waitForSFSTurboStatus(sfsClient *golangsdk.ServiceClient, shareId string) resource.StateRefreshFunc {
+func waitForSFSTurboStatus(sfsClient *golangsdk.ServiceClient, shareId string, isDelete bool) resource.StateRefreshFunc {
```

Comment:
It is best to split this waitFor method into two scenarios and use them respectively for create and delete.